content/_releases/22/28.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Product Update - v22.28
+description: Details of the product update v22.28 from 14th of July 2022.
+layout: rel
+version: "22.28"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-07-14
+redirect_from:
+  - /releases/22.28.html
+---
+
+## Improvements and Updates
+
+### Per-tenant frontend styles feature removed
+
+We've removed feature that builds per-tenant styles and applied them based on `tenant.cssEnabled` and `tenant.code property`. We also removed all tenant styles from our codebase.
+
+> PLease visit our [API documentation]({{site.data.tenant.apiBaseUri}}/docs/v2/#tenants) for more information.
+
+###  Remove Mailchimp related code
+
+Since we no longer support Mailchimp service we have removed everything related to Mailchimp from platform.
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements included in this release.
+
+###   Add Ability to Set Per-Service Custom Ingress Annotations
+
+To increase flexibility in ingress configurations for different hosts we made a **breaking change**. We have changed the structure of model property `tenant.ingressConfig.annotations` and `helm.Values.global.services.handmaiden.annotations`:
```

Comment:
of **the** model propert**ies**

Please be aware that upgrading to elastic.io Tenants to this latest version requires the small changes described below: